File: tests/indexing/parsing/lang/fixtures/rust/Comments.rs
/// Adds two numbers together.
/** Block doc comment describing the function. */
pub fn with_comments(a: i32, b: i32) -> i32 {
    // line comment
    let sum = a + b; /* trailing block comment */
    let url = "http://not-a-comment";
    let _ = url;
    sum
}

File: tests/indexing/parsing/lang/fixtures/php/Comments.php
<?php

class Documented
{
    /**
     * Adds two numbers together.
     */
    public function withComments(int $a, int $b): int
    {
        // line comment
        $sum = $a + $b; # hash comment
        /* block comment */
        $url = "http://not-a-comment";
        return $sum;
    }
}
